NodeBase/Dockerfile

```diff
@@ -95,6 +95,7 @@ COPY selenium.conf /etc/supervisor/conf.d/
 ENV SCREEN_WIDTH 1360
 ENV SCREEN_HEIGHT 1020
 ENV SCREEN_DEPTH 24
+ENV SCREEN_DPI 96
```

Comment:
Please only modify the `Dockerfile.txt` file.

Reply:
Oops, should be fixed now!